storage/feed.go

```diff
@@ -79,6 +79,33 @@ type StartStoreEvent struct {
 	StoreID proto.StoreID
 }
 
+// StoreStatusEvent contains the current descriptor for the given store.
+//
+// Because the descriptor contains information that cannot currently be computed
+// from other events, this event should be periodically broadcast by the store
+// independently of other operations.
+type StoreStatusEvent struct {
+	Desc *proto.StoreDescriptor
```

Comment:
this is still not being used, can you remove it? it seems much more natural to me to introduce this in the same commit that actually uses it.